### PR TITLE
EZP-27010: Allow to configure UDW confirm label button

### PR DESCRIPTION
**Pull request by StephaneDiot**
jira : https://jira.ez.no/browse/EZP-27010

## Description

Goal was to add a parameter to the 'contentDiscover' event (event which fires the UDW) to allow to configure the label of the confirm button. This parameter is named 'confirmLabel'

## Tests

Unit and manually tested

### CODE REVIEW

File: Tests/js/views/services/ez-discoverybarviewservice.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 <script type="text/javascript" src="../../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
 <script type="text/javascript" src="./assets/ez-discoverybarviewservice-tests.js"></script>
 <script type="text/javascript" src="../../assets/ez-translator.js"></script>
+
```
Comment (dpobel) [inline Tests/js/views/services/ez-discoverybarviewservice.html]: not sure the most useful line in the world :)

**Comment by StephaneDiot** [comment]
updated @dpobel

File: Resources/translations/contentedit.en.xlf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2017-01-31T10:50:08Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
+  <file date="2017-03-07T15:55:42Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
```
Comment (dpobel) [inline Resources/translations/contentedit.en.xlf]: please include this kind of change only in the files that actually change.

**Comment by StephaneDiot** [comment]
updated @dpobel

File: Resources/public/js/views/services/ez-discoverybarviewservice.js [hunk 3]
```diff
@@ -20,10 +20,62 @@ YUI.add('ez-discoverybarviewservice', function (Y) {
...
+        /**
+         * Navigate to the selected location.
+         *
+         * @method _navigateToLocation
+         * @param {eventFacade} e
+         * @protected
+         */
+        _navigateToLocation: function (e) {
+            this.get('app').navigateTo('viewLocation', {
+                    id: e.selection.location.get('id'),
```
Comment (yannickroger) [inline Resources/public/js/views/services/ez-discoverybarviewservice.js]: too much indent

File: Resources/public/js/views/services/ez-universaldiscoveryviewservice.js [hunk 4]
```diff
@@ -96,7 +96,17 @@ YUI.add('ez-universaldiscoveryviewservice', function (Y) {
         },
     }, {
         ATTRS: {
-
+            /**
+             *
```
Comment (yannickroger) [inline Resources/public/js/views/services/ez-universaldiscoveryviewservice.js]: comment ?

File: Tests/js/views/services/assets/ez-discoverybarviewservice-tests.js [hunk 5]
```diff
@@ -59,6 +59,7 @@ YUI.add('ez-discoverybarviewservice-tests', function (Y) {
         },
 
         tearDown: function () {
+
```
Comment (yannickroger) [inline Tests/js/views/services/assets/ez-discoverybarviewservice-tests.js]: nitpick: useless

File: Resources/public/js/views/services/ez-discoverybarviewservice.js [hunk 6]
```diff
@@ -20,10 +20,62 @@ YUI.add('ez-discoverybarviewservice', function (Y) {
...
+                rootDepth;
+
+            if (this._isLocationViewDisplayed()) {
+               startingLocationId = this.get('app').get('activeView').get('location').get('id');
+               rootDepth = 1;
+            }
+
+            this.fire('contentDiscover', {
+                config: {
+                    title: Y.eZ.trans('universaldiscovery.content.browser', {}, 'universaldiscovery'),
```
Comment (dpobel) [inline Resources/public/js/views/services/ez-discoverybarviewservice.js]: `universaldiscovery` is not the right context for this string, `bar` would be more suited.

**Pull request merged**